base_eeg/test.py

```diff
@@ -0,0 +1,26 @@
+import mne
+import pathlib
+import autoreject as ar
+from mne_bids import BIDSPath, read_raw_bids
+import finalRej as fr
+    
+def main():
+    # import raw data 
+    bids_root = pathlib.Path('../raw_data/eeg_matchingpennies')
+    bids_path = BIDSPath(subject='05', task='matchingpennies', datatype='eeg', root=bids_root)
+    raw = read_raw_bids(bids_path)
+    # Generate events to segment data into epochs 
+    events, event_id = mne.events_from_annotations(raw)
+    epochs = mne.Epochs(raw, events, event_id=event_id, preload=True)
+    # Set montage
+    easy_montage = mne.channels.make_standard_montage('standard_1020')
+    epochs.set_montage(easy_montage)
+
+    #Sample user parameter 
+    user_params={'tmin':-0.2,'tmax':0.5}
+    output_dict_finalRej=fr.final_reject_epoch(user_params, epochs, raw)
+    print('this is the the output dictionary ---> ',output_dict_finalRej)
+
+if __name__=='__main__':
+    main()
+
```

Comment:
final_reject works great for matching_pennies, nice work on annotations
```
154 bad epochs dropped
defaultdict(<class 'dict'>, {
'interpolatedChannels': 
    ['C3', 'CP5', 'CP1', 'C4', 'CP2', 'CP6'], 
'epochsDropped': 
    {'FC5': '0', 'FC1': '0', 'C3': '11', 'CP5': '79', 'CP1': '33', 'FC2': '0', 'FC6': '0', 'C4': '32', 'CP2': '54', 'CP6': '154'}
})
```

Final_reject fails for new BIDS data from feat-io. New BIDS data might need to be looked at closer:

```
ValueError: No matching events found for 82 (event id 10004)
```